r-devel/Dockerfile

```diff
@@ -69,8 +69,9 @@ ENV LANG=en_US.UTF-8
 RUN update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 800 \
     --slave /usr/bin/g++ g++ /usr/bin/g++-8
 
-RUN update-alternatives --install /usr/bin/clang clang /usr/bin/clang-7 800 \
-    --slave /usr/bin/clang++ clang++ /usr/bin/clang++-7
+RUN update-alternatives --install /usr/bin/clang clang /usr/bin/clang-7 700 \
```

Comment:
I'm not sure if the `700` is relevant

Reply:
I think that is a priority level, so I think you can just leave it at the original value.